app/models/npq_profile.rb

```diff
@@ -10,4 +10,11 @@ class NpqProfile < ApplicationRecord
     yes_in_first_two_years: "yes_in_first_two_years",
     yes_over_two_years: "yes_over_two_years",
   }
+
+  enum funding_choice: {
+    school: "school",
+    trust: "trust",
+    self: "self",
+    another: "another",
```

Comment:
If it's another should that 'another choice' be stored in another field? `funding_choice_other`?

Reply:
There's no expandy box thing and is just a radio button. My assumption is the provider would get back in touch with the user or would have to double ask the user for this info. They have to go thru another registration process on the provider page anyway

![Screenshot 2021-06-25 at 16 10 24](https://user-images.githubusercontent.com/92580/123445566-ea40e480-d5cf-11eb-9e56-d64502f4f288.png)